presto-main/src/main/java/com/facebook/presto/server/remotetask/HttpRemoteTask.java

```diff
@@ -367,10 +361,10 @@ public HttpRemoteTask(
                     initialTask.getTaskStatus(),
                     taskStatusRefreshMaxWait,
                     taskStatusCodec,
-                    executor,
+                    taskEventLoop,
```

Comment:
Maybe just pass it once and call it `taskEventLoop`

Reply:
Sure. I was trying not to modify the code in other files and keep this pr straightforward but looks like TaskInfoFetcher and ContinuousTaskStatusFetcher are only being used here. Let me update them as well.